.buildkite/pipeline.yml

```
@@ -4,375 +4,437 @@ agents:
-      system: ${macos}
-
-  - label: 'Build integration tests (macOS, arm64)'
-    key: macos-arm64-tests-build-integration
-    depends_on: macos-nix
-    command: nix build -L .#packages.aarch64-darwin.integration-exe
-    agents:
-      system: ${macos}
-      queue: "cardano-wallet"
-
-  - label: 'Build package (macOS, x86_64)'
-    key: macos-intel-package
-    depends_on: macos-nix
-    command: 'nix build -o result/macos-intel .#packages.x86_64-darwin.ci.artifacts.macos-intel.release'
-    artifact_paths: [ "./result/macos-intel/**" ]
-    agents:
-      system: ${macos}
-
-  - label: 'Build package (macOS, arm64)'
-    key: macos-arm64-package
-    depends_on: macos-nix
-    command: 'nix build -o result/macos-silicon .#packages.aarch64-darwin.ci.artifacts.macos-silicon.release'
-    artifact_paths: [ "./result/macos-silicon/**" ]
-    agents:
-      system: ${macos}
-
-  - block: "Build package (linux)"
-    if: |
-      build.branch !~ /^gh-readonly-queue\/master/
-        && build.branch != "master"
-        && build.branch !~ /^release-candidate/
-        && build.branch != "rc-latest"
-    depends_on: linux-nix
-    key: trigger-build-linux-package
-
-  - label: 'Build package (linux)'
-    key: linux-package
-    depends_on: [linux-nix, trigger-build-linux-package]
-    command: nix build -o result/linux .#ci.artifacts.linux64.release
-    artifact_paths: [ "./result/linux/**" ]
-    agents:
-      system: ${linux}
-    env:
-      TMPDIR: "/cache"
-
-  - block: "Build windows artifacts"
-    if: |
-      build.branch !~ /^gh-readonly-queue\/master/
-        && build.branch != "master"
-        && build.branch !~ /^release-candidate/
-        && build.branch != "rc-latest"
-    depends_on: linux-nix
-    key: trigger-build-windows-artifacts
-
-  - label: 'Build package (windows)'
-    key: windows-package
-    depends_on: [linux-nix, trigger-build-windows-artifacts]
-    command: nix build -o result/windows .#ci.artifacts.win64.release
-    artifact_paths: [ "./result/windows/**" ]
-    agents:
-      system: ${linux}
-    env:
-      TMPDIR: "/cache"
-
-  - label: 'Build testing bundle (windows)'
-    key: windows-testing-bundle
-    depends_on: [linux-nix, trigger-build-windows-artifacts]
-    command: nix build -o result/windows-tests .#ci.artifacts.win64.tests
-    artifact_paths: [ "./result/windows-tests/**" ]
-    agents:
-      system: ${linux}
-    env:
-      TMPDIR: "/cache"
-
-  - block: "Run E2E tests"
-    if: |
-      build.branch !~ /^gh-readonly-queue\/master/
-        && build.branch != "master"
-        && build.branch !~ /^release-candidate/
-        && build.branch != "rc-latest"
-    depends_on: linux-nix
-    key: trigger-e2e-tests
-
-  - label: "Run E2E tests"
-    key: e2e
-    depends_on: trigger-e2e-tests
-    command: 'nix develop --command bash -c "just e2e-local"'
-    agents:
-      system: ${linux}
-    env:
-      TMPDIR: "/cache"
-
-  - label: Manage rc-latest tags
+  - group: "Linux Setup"
```

```suggestion
  - group: "Linux Artifacts"
```

Could you adjust to the nomenclature of "Continuous Integration :: AP"? (Same for the other groups.)